addon-test-support/@ember/test-helpers/dom/click.ts

```diff
@@ -33,6 +33,7 @@ export function __click__(
   element: Element | Document | Window,
   options: MouseEventInit
 ): void {
+  options = { ...options, detail: 1 };
```

Comment:
From https://github.com/emberjs/ember-test-helpers/blob/master/API.md#click

I believe this would be a breaking change as users could no longer override `detail` on the event.

Reply:
That is a good point.

Reply:
ya, you'd have to do it the other way around (so the users provided `detail` option "wins"):

```suggestion
  options = { detail: 1, ...options};
```